toolkit.pro

```diff
@@ -0,0 +1,34 @@
+################################################################################
```

Comment:
Do we want this? The folders referred to in OTHER_FILES look to reference the old pattern